### ISSUE TITLE
Accessibility issue when NavigationView.PaneDisplayMode="Top"

### ISSUE THREAD

**Issue by mrlacey** [microsoft/microsoft-ui-xaml#680]
**Describe the bug**

When `PaneDisplayMode` is set to `Top`, `AutomationProperties.PositionInSet` and `AutomationProperties.SizeOfSet` are not specified which is a failure of **Section 508 502.3.1**

This doesn't happen when the Pane is on the left.
(Although it does also happen when the pane is on the left and  there are no Menu Items defined, but having the navigation view with no items in the pane is an edge case.)

**Steps to reproduce the bug**

1. Create an empty app with the following in the main page

```
    <NavigationView IsSettingsVisible="True" PaneDisplayMode="Top">
        <NavigationView.MenuItems>
            <NavigationViewItem Content="One" Icon="Accept" />
        </NavigationView.MenuItems>
    </NavigationView>
```

2. Add latest NuGet package (number below)
3. Run the app.
4. Test with [Accessibility Insights for Windows](https://accessibilityinsights.io/)

**Expected behavior**

Should not have errors (test fail) reported as these values should be **1**.

Failure occurs even though the code ([here](https://github.com/microsoft/microsoft-ui-xaml/blob/2a0b7bc007bcf62cc8eae670071c2d3a4a9ce55f/dev/NavigationView/NavigationViewItemAutomationPeer.cpp#L70) and [here](https://github.com/microsoft/microsoft-ui-xaml/blob/2a0b7bc007bcf62cc8eae670071c2d3a4a9ce55f/dev/NavigationView/NavigationViewItemAutomationPeer.cpp#L96)) indicates that the expected values should be returned. 

**Screenshots**

![image](https://user-images.githubusercontent.com/189547/57485627-0acb4a80-7261-11e9-9141-2038af097bc2.png)

**Version Info**

NuGet package version: 

2.2.190430003-prerelease

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (xxxxx)                   |    |
| October 2018 Update (17763)   | Yes   |
| April 2018 Update (17134)        |    |
| Fall Creators Update (16299)     |    |
| Creators Update (15063)           |    |
| Anniversary Update (14393)      |    |

| Device form factor | Saw the problem? |
| :-------------------- | :------------------- |
| Desktop                 | Yes   |
| Mobile                   |    |
| Xbox                      |    |
| Surface Hub          |     |
| IoT                         |     |

**Additional context**

**Comment by YuliKl** [comment]
@mrlacey - does this error occur when more than one item get added? I'm not seeing the issue in the Controls Gallery app, but the app show four menu items in each of its top nav samples.

**Comment by mrlacey** [comment]
@YuliKl yes it happens with more than one item.

In the app I just have this:

```xaml
     <NavigationView IsSettingsVisible="True" PaneDisplayMode="Top">
        <NavigationView.MenuItems>
            <NavigationViewItem Content="One" Icon="Accept" />
            <NavigationViewItem Content="Two" Icon="Account" />
            <NavigationViewItem Content="Three" Icon="AddFriend" />
            <NavigationViewItem Content="Four" Icon="Admin" />
        </NavigationView.MenuItems>
    </NavigationView>
```

But Accessibility Insights finds an error

![image](https://user-images.githubusercontent.com/189547/57587675-8c1f1900-74bd-11e9-8c3a-f10829a38198.png)

And Inspect shows the properties not set

![image](https://user-images.githubusercontent.com/189547/57587745-c6d58100-74be-11e9-8484-f6b455b6e9fa.png)

I've double checked I'm using version **2.2.190430003-prerelease**


When I test against the gallery app (v1.2.7.0) the values are set correctly. 
The gallery app uses v**2.1.190405004** so I set my app to this and the issue still occurs.
I've also tried putting the NavigationView inside other content but that doesn't fix it either. :(

**Comment by chingucoding** [comment]
I think this was fixed with one of changes made to the NavigationView in the past few months, so we can probably close this.

**Comment by ranjeshj** [comment]
PositionInSet and SizeInSet seem to be working now with latest prerelease.
